lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -258,9 +260,13 @@ def transform_mailing_address(veteran, veteran_identification)
           veteran_identification.mailing_address.apartment_or_unit_number +=
             veteran['currentMailingAddress']['addressLine3']
         end
-        veteran_identification.mailing_address.city = veteran['currentMailingAddress']['city']
-        veteran_identification.mailing_address.state = veteran['currentMailingAddress']['state']
-        veteran_identification.mailing_address.zip_first_five = veteran['currentMailingAddress']['zipFirstFive']
+        veteran_identification.mailing_address.city = veteran['currentMailingAddress']['militaryPostOfficeTypeCode'] ||
+                                                      veteran['currentMailingAddress']['city']
+        veteran_identification.mailing_address.state = veteran['currentMailingAddress']['militaryStateCode'] ||
+                                                       veteran['currentMailingAddress']['state']
+        veteran_identification.mailing_address.zip_first_five =
+          veteran['currentMailingAddress']['internationalPostalCode'] ||
+          veteran['currentMailingAddress']['zipFirstFive']
```

Comment:
Excuse my ignorance. Just double checking, is there a possibility where a Veteran could potentially list both a military address and a civilian address? 

And if so, do we have an official way to know which takes priority? (e.g. military always priority, most recent takes priority, etc.)

Reply:
It is _not_ possible. What gets submitted is one address (current mailing address and/or  change of address), and the transform code in data_transformation_all_claims adds new military/intl elements to the JSON based on the state and country submitted. We send these to EVSS as-is, but as it turns out (per Michael Harlow) EVSS is simply filling those elements into one address, anyway. So for LH, we're just doing that on _our_ end now. If militaryPostOfficeTypeCode, use it for city instead, and so for militaryStateCode + state and internationalPostalCode + zipFirstFive